src/Serilog.Extensions.Logging/Serilog.Extensions.Logging.csproj

```diff
@@ -31,7 +31,7 @@
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net461' or '$(TargetFramework)' == 'netstandard2.0' ">
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.*" />
```

Comment:
note that this wont effect the resulting package. on each build the resultant nuget will have the current release version of Microsoft.Extensions.Logging hard coded as a package dependency

Reply:
Argh, thanks Simon - lost track of this particular footgun :-)

Reply:
I only remember due to my missing foot

Reply:
😂